### PR TITLE
[CloudWatchLogs] Remove unnecessary sequenceToken from log events data

### PR DESCRIPTION
**Pull request by Dukecz**
Fixes https://github.com/async-aws/aws/issues/1363

### CODE REVIEW

**Comment by jderusse** [comment]
LGTM: Could you please look at the failing tests

**Comment by Dukecz** [comment]
@jderusse fixed, might be worth rebasing into one commit. I have been editing it in codespace and it has a lot to be desired.

File: src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php [hunk 1]
```diff
@@ -234,22 +228,6 @@ private function getMessageSize($record): int
...
-            ->getLogStreams(true);
-
-        /** @var LogStream $stream */
-        foreach ($existingStreams as $stream) {
-            if ($stream->getLogStreamName() === $this->options['stream'] && $stream->getUploadSequenceToken()) {
-                $this->sequenceToken = $stream->getUploadSequenceToken();
-            }
-        }
-
         $this->initialized = true;
```
Comment (jderusse) [inline src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php:231]: I believe we don't need this method anymore

File: src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php [hunk 2]
```diff
@@ -281,13 +259,10 @@ private function send(array $entries): void
             'logGroupName' => $this->options['group'],
             'logStreamName' => $this->options['stream'],
             'logEvents' => $entries,
-            'sequenceToken' => $this->sequenceToken,
         ];
 
         $this->checkThrottle();
 
         $response = $this->client->putLogEvents($data);
```
Comment (jderusse) [inline src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php:266]: `$response` variable is not needed anymore and can be removed

**Review by jderusse — COMMENTED** [review]
Thank yoiu for working on this.
I left 2 minors comments

**Comment by jderusse** [comment]
thank you @Dukecz

**Pull request merged**